src/com/mendix/widget/carousel/components/__tests__/CarouselItem.spec.ts

```diff
@@ -1,23 +1,33 @@
 import { ShallowWrapper, shallow } from "enzyme";
-import { DOM, createElement } from "react";
+import { createElement } from "react";
 
 import { CarouselItem, CarouselItemProps } from "../CarouselItem";
 
 describe("CarouselItem", () => {
     const url = "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7";
-    let wrapper: ShallowWrapper<CarouselItemProps, any>;
-    beforeEach(() => wrapper = shallow(createElement(CarouselItem, { url, active: true })));
+    let output: ShallowWrapper<CarouselItemProps, any>;
 
-    it("should be active", () => {
-        expect(wrapper).toBeElement(
-            DOM.div({ className: "item active" }, DOM.img({ alt: "Carousel image", src: url }))
-        );
+    beforeEach(() => output = shallow(createElement(CarouselItem, { url, active: true })));
+
+    it("should be a div with one img child", () => {
+        expect(output.type()).toBe("div");
+        expect(output.children().length).toBe(1);
+        expect(output.children().first().type()).toBe("img");
+    });
+
+    it("should be active as set in the props", () => {
```

Comment:
=> "should add a css class if active"

The fact that that happens through a prop is an implementation detail.